sphinx/source/docs/quickstart.rst

```diff
@@ -6,374 +6,585 @@ Quickstart
+.. _Bokeh for R: http://hafen.github.io/rbokeh/
+.. _Bokeh for Scala: https://github.com/bokeh/bokeh-scala
+.. _Bokeh GitHub repository: https://github.com/bokeh/bokeh
+.. _Bokeh mailing list: https://groups.google.com/a/continuum.io/forum/#!forum/bokeh
+.. _Bokeh NBViewer Gallery: http://nbviewer.ipython.org/github/bokeh/bokeh-notebooks/blob/master/index.ipynb
+.. _interact_basic: https://github.com/bokeh/bokeh/blob/master/examples/plotting/notebook/interact_basic.ipynb
+.. _interact_numba: https://github.com/bokeh/bokeh/blob/master/examples/plotting/notebook/interact_numba.ipynb
+.. _Twitter: http://twitter.com/BokehPlots
+.. _Vine: https://vine.co/bokehplots
+.. _YouTube: https://www.youtube.com/channel/UCK0rSk29mmg4UT4bIOvPYhw
+
+
+.. |bokeh.charts| replace:: :ref:`bokeh.charts <bokeh.charts>`
+.. |bokeh.models| replace:: :ref:`bokeh.models <bokeh.models>`
+.. |bokeh.plotting| replace:: :ref:`bokeh.plotting <bokeh.plotting>`
+
+.. |glyphs| replace:: ref:`glyphs <bokeh.models.glyphs>`
+.. |markers| replace:: ref:`markers <bokeh.models.markers>`
+
+.. |figure| replace:: :func:`figure <bokeh.plotting.figure>`
+.. |Figure| replace:: :class:`Figure <bokeh.plotting.Figure>`
+
+.. |legend| replace:: :class:`legend <bokeh.plotting.Figure.legend>`
+.. |grid| replace:: :class:`grid <bokeh.plotting.Figure.grid>`
+.. |xgrid| replace:: :class:`xgrid <bokeh.plotting.Figure.xgrid>`
+.. |ygrid| replace:: :class:`ygrid <bokeh.plotting.Figure.ygrid>`
+.. |axis| replace:: :class:`axis <bokeh.plotting.Figure.axis>`
+.. |xaxis| replace:: :class:`xaxis <bokeh.plotting.Figure.xaxis>`
+.. |yaxis| replace:: :class:`yaxis <bokeh.plotting.Figure.yaxis>`
+
+.. |output_file| replace:: :func:`output_file <bokeh.io.output_file>`
+.. |output_notebook| replace:: :func:`output_notebook <bokeh.io.output_notebook>`
+.. |output_server| replace:: :func:`output_server <bokeh.io.output_server>`
+.. |save| replace:: :func:`save <bokeh.io.save>`
+.. |show| replace:: :func:`show <bokeh.io.show>`
+
+.. |ColumnDataSource| replace:: :class:`ColumnDataSource <bokeh.models.sources.ColumnDataSource>`
+.. |DatetimeAxis| replace:: :class:`DatetimeAxis <bokeh.models.axes.DatetimeAxis>`
+
+.. |Line| replace:: :class:`Line <bokeh.models.glyphs.Line>`
+.. |Figure.circle| replace:: :func:`Figure.line <bokeh.plotting.Figure.circle>`
+.. |Figure.line| replace:: :func:`Figure.line <bokeh.plotting.Figure.line>`
+.. |Figure.square| replace:: :func:`Figure.square <bokeh.plotting.Figure.square>`
+.. |Figure.triangle| replace:: :func:`Figure.triangle <bokeh.plotting.Figure.triangle>`
+
+.. |gridplot| replace:: :func:`gridplot <bokeh.io.gridplot>`
+.. |hplot| replace:: :func:`hplot <bokeh.io.hplot>`
+.. |vplot| replace:: :func:`vplot <bokeh.io.vplot>`
+
+.. |line_props| replace:: :class:`Line Properties <bokeh.mixins.LineProps>`
+.. |fill_props| replace:: :class:`Fill Properties <bokeh.mixins.FillProps>`
+.. |text_props| replace:: :class:`Text Properties <bokeh.mixins.TextProps>`
 
 
```

Comment:
Overall suggestion about live examples.. what about turning the wheel_zoom of for those? It's really annoying when scrolling the page up and down... most of the times the figure captures the mouse pointer and messes up with figure..